src/views/bibliography_item.tsx

```diff
@@ -100,7 +148,9 @@ export class BibliographyItemView<PropsType extends BibliographyItemProps>
           )
         })
 
-        // TODO:: add event listener for edit button
+        editButton.addEventListener('click', () => {
+          this.showPopper(this.node.attrs.id)
```

Comment:
add `this.popperContainer = undefined;` so the second time we render popper again.